Deck.php

```diff
@@ -0,0 +1,179 @@
+<?php
+
+class Deck
+{
+
+    // Properties
+    private int $id;
+    private string $name;
+    private string $format;
+    private string $colourID;
+    private string $archetype;
+    private string $last_updated;
+    private string $primer;
+    private string $image;
+    private string $moxfield_link;
+
+
+    public function __construct(
+        string $name, string $format, string $colourID, string $archetype, string $last_updated
+        , string $primer, string $image, string $moxfield_link, int $id = -1
+    )
+    {
+        $this->id = $id;
+        $this->name = $name;
+        $this->format = $format;
+        $this->colourID = $colourID;
+        $this->archetype = $archetype;
+        $this->last_updated = $last_updated;
+        $this->primer = $primer;
+        $this->image = $image;
+        $this->moxfield_link = $moxfield_link;
+
+    }
+
+    /**
+     * @return int
+     */
```

Comment:
You don't really need these comments. The method signature (` public function getId(): int`) already has all of the information needed, so having these comments is really just an extra thing to look after.

I know phpstorm adds them by default, but now that PHP has full typehinting support, we really don't need them (unless the method does something that can't be properly explained by the signature)

I won't make you remove them, since it's super nit picky, just something to think about!

Reply:
Removed them all - good to know not needed!